packages/terra-list/src/ListSectionHeader.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Arrange from 'terra-arrange';
+import styles from './ListSectionHeader.scss';
+
+const cx = classNames.bind(styles);
+
+const KEYCODES = {
+  ENTER: 13,
+  SPACE: 32,
+};
+
+const propTypes = {
+  /**
+   * Text to be displayed on the ListSectionHeader.
+   */
+  title: PropTypes.string.isRequired,
```

Comment:
Ya, As per the Zeplin designs, ListSectionHeader is going to have props for title, count, notificationIcon and few others. Its for separating the Count of List Items in a section and the Header part itself. [Zeplin](https://app.zeplin.io/project/5a8c5310e635140e3cc134c2/screen/5a994631fa6d117509f784fb)

Reply:
I would think, if we only want to display the count, then maybe we can get rid of the title prop and not mark it as isRequired. Also, since count and other props are not implemented, I thought we could leave it as Required, since that is how our team (MAR) uses it to display medication information as of now. 

However, I might not be the correct person to answer this. @neilpfeiffer or @PaulSolDev Any idea on this?

Reply:
Let's make this prop optional.